cfbs/masterfiles/generate_vcf_git_checkout.py

```diff
@@ -0,0 +1,92 @@
+import os
+import shutil
+import subprocess
+import sys
+
+from cfbs.git import git_exists
+from cfbs.utils import write_json
+from cfbs.masterfiles.analyze import (
+    finalize_vcf,
+    initialize_vcf,
+    versions_checksums_files,
+)
+
+DIR_PATH = "."
+"""The path of the working directory."""
+
+MPF_URL = "https://github.com/cfengine/masterfiles"
+MPF_PATH = os.path.join(DIR_PATH, "masterfiles")
+
+
+def generate_vcf_git_checkout(interesting_tags=None):
+    # clone the MPF repo every time the script is run, in case there are updates
+    if os.path.isdir(MPF_PATH):
+        shutil.rmtree(MPF_PATH)
+
+    subprocess.run(
+        ["git", "clone", MPF_URL],
+        cwd=DIR_PATH,
+        check=True,
+    )
+
+    if not git_exists():
```

Comment:
Why is the check for git exists after a use of git? Seems like it should be up around line 25.

Reply:
Oops, that's an oversight, in an earlier version checking in this place was sufficient. Will fix.

Reply:
Fixed in https://github.com/cfengine/cfbs/pull/208/commits/b516b19c038d149e10df342e60b71cb777cc34b5